PANDORA/Pandora/Modelling_functions.py

```diff
@@ -224,134 +231,142 @@ def predict_anchors_netMHCpan(peptide, allele_type,
     Returns: (lst): list of predicted anchor predictions
 
     '''
-    all_netMHCpan_alleles = []
-    with open(PANDORA.PANDORA_path + '/../netMHCpan-4.1/data/allelenames') as f:
-        for line in f:
-            all_netMHCpan_alleles.append(line.split(' ')[0])#.replace(':',''))
-    
-    ## Format alleles
-    target_alleles = [i.replace('*','') for i in allele_type]
-    ## Make sure only netMHCpan available alleles are used
-    target_alleles = [i for i in target_alleles if i in all_netMHCpan_alleles]
-    
-    if len(target_alleles) == 0:
-        print('ERROR: The provided Target allele is not available in NetMHCpan-4.1')
-        return None
     
-    target_alleles_str = ','.join(target_alleles)
-    
-    # Setup files
-    netmhcpan = PANDORA.PANDORA_path + '/../netMHCpan-4.1/netMHCpan'
-    infile = PANDORA.PANDORA_path + '/../netMHCpan-4.1/tmp/%s_%s_%s.txt' %(
-        peptide, target_alleles[0].replace('*','').replace(':',''), datetime.today().strftime('%Y%m%d_%H%M%S'))
-    outfile = PANDORA.PANDORA_path + '/../netMHCpan-4.1/tmp/%s_%s_%s_prediction.txt' %(
-        peptide, target_alleles[0].replace(':',''), datetime.today().strftime('%Y%m%d_%H%M%S'))
-    
-    # Write peptide sequence to input file for netMHCIIpan
-    with open(infile, 'w') as f:
-        f.write(peptide)
-    
-    os.system('%s -p %s -a %s > %s' %(netmhcpan, infile, target_alleles_str, outfile))
-    
-    # Get the output from the netMHCIIpan prediction
-    # {allele: (core, %rank_EL)}
-    pred = {}
-    with open(outfile) as f:
-        for line in f:
-            if peptide in line and not line.startswith('#'):
-                ln = [i for i in line[:-1].split(' ') if i != '']
-                #ln[3] is core, ln[9] is Icore
-                try:
-                    pred[ln[1]].append((ln[3], float(ln[12])))
-                except KeyError:
-                    pred[ln[1]] = [(ln[3], float(ln[12]))]
+    # Retrieves the enviroment of the netMHCpan script and the path of the directory where this script is in 
```

Comment:
Please fix this comment. This line does not "retrieve the environment of netMHCpan", it retrieves the environment variable netMHCpan, which is a path.

Reply:
I did this in commit [2c170ad](https://github.com/X-lab-3D/PANDORA/pull/196/commits/2c170ad49032e2083b5f232303f79e3afeb1ac5b)